pages.es/linux/yum.md

```diff
@@ -0,0 +1,28 @@
+# yum
+
+> Administrador de paquetes para RHEL, CentOS y Fedora (para versiones anteriores).
+> Más información: <https://manned.org/yum>.
```

Comment:
```suggestion
> Más información: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```

We prefer manned.org in a general case but the [original page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/yum.md) uses another manual source so it's better to use the same one I think.